### PR TITLE
V.1.14.0 + cleaning up

### PR DESCRIPTION
**Pull request by ericgaspar**
## Problem
- *Not up to date with upstream*

## Solution
- *Upgrade to 1.14.0*
- *Install PHP extentions*
- *set PHP version as a variable*
- *Add more language to choose from in manifest*
- *Cleaning up*

## PR Status
- [ ] Code finished.
- [ ] Tested with Package_check.
- [ ] Fix or enhancement tested.
- [ ] Upgrade from last version tested.
- [ ] Can be reviewed and tested.

## Validation
---
[Build Status/badge/icon)](https://ci-apps-dev.yunohost.org/jenkins/job/rainloop_ynh%20PR68%20(ericgaspar)/)

### CODE REVIEW

File: conf/data/configs/application.ini [hunk 1]
```diff
@@ -79,8 +81,10 @@ admin_password = "12345"
 allow_admin_panel = On
 allow_two_factor_auth = Off
 force_two_factor_auth = Off
-allow_universal_login = Off
+hide_x_mailer_header = Off
 admin_panel_host = ""
+admin_panel_key = "admin"
```
Comment (alexAubin) [inline conf/data/configs/application.ini:86]: Uuuh, is this like the password to access some sort of admin interface o.O ?

**Review by alexAubin — APPROVED** [review]
LGTM, thanks for all the work ! :tada: :+1:

File: conf/data/configs/application.ini (hunk 1 above)
Comment (ericgaspar) [reply conf/data/configs/application.ini:86]: As I understand, it is a layer of protection to protect the admin panel.
By default, the administration panel is accessible with this address: `http://DOMAIN.TLD/rainloop/app/?admin`
if you set `admin_panel_key = "admin"` to `admin_panel_key = "my-secret-admin"`, the admin panel will be accessible with the following address:  `http://DOMAIN.TLD/rainloop/app/?my-secret-admin`
cf. https://github.com/RainLoop/rainloop-webmail/issues/976

File: scripts/_common.sh [hunk 2]
```diff
@@ -4,6 +4,10 @@
 # PERSONAL HELPERS
 #=================================================
 
+pkg_dependencies="php${YNH_PHP_VERSION}-json php${YNH_PHP_VERSION}-curl php${YNH_PHP_VERSION}-xml php${YNH_PHP_VERSION}-dom"
+
+YNH_PHP_VERSION="7.3"
```
Comment (kay0u) [inline scripts/_common.sh]:
```suggestion
YNH_PHP_VERSION="7.3"

pkg_dependencies="php${YNH_PHP_VERSION}-json php${YNH_PHP_VERSION}-curl php${YNH_PHP_VERSION}-xml php${YNH_PHP_VERSION}-dom"
```

**Pull request merged**